aidants_connect_web/views/service.py

```diff
@@ -41,6 +41,7 @@ def home_page(request):
 @login_required
 def logout_page(request):
     logout(request)
+    django_messages.success(request, "Vous venez de vous déconnecter avec succès.")
```

Comment:
```suggestion
    django_messages.success(request, "Vous êtes maintenant déconnecté·e.")
```

(j'ai l'impression que les tournures genre _avec succès_ ça fait traduction un peu forcée de l'anglais _successfully_, mais qu'en français ça n'est pas très idiomatique)

Reply:
j'ai fait la modif pour ce message là, mais j'ai laissé les 2-3 autres `avec succès` dans le code. Ca ne me choque pas cette tournure :)